mesos-rxjava-core/src/main/java/org/apache/mesos/rx/java/Rx.java

```diff
@@ -0,0 +1,20 @@
+package org.apache.mesos.rx.java;
+
+import org.jetbrains.annotations.NotNull;
+import rx.Scheduler;
+import rx.schedulers.Schedulers;
+
+public final class Rx {
```

Comment:
Correct, I was using it for debugging. I figure we'll come up with a few more utilities in the future and we can be more formal then.